### DISCUSSION TITLE
How to switch between tab.

### DISCUSSION THREAD

**Discussion by anshiq** [LunarVim/LunarVim#4066] [General]
how can i switch between opened tabs of files in lunar vim

**Comment by brneor** [comment]
`<leader> b b` `<leader> b n` `<leader> b j`

**Reply by anshiq** [reply]
so i want to change this keybinding to shift + h and shift + k how can i do that

**Reply by brneor** [reply]
```lua
lvim.keys.normal_mode["K"] = ":BufferLineCycleNext<CR>"
lvim.keys.normal_mode["H"] = ":BufferLineCyclePrev<CR>"
```

**Reply by cassepipe** [reply]
> so i want to change this keybinding to shift + h and shift + k how can i do that

Try  to read the commented parts of the default config. Those two line were already there, commented out.

**Reply by Luciomelo1407** [reply]
using the default keybinding how can I close a tab?

**Reply by brneor** [reply]
`<leader> c`

You can see all keybindings with `<leader> L k`

**Reply by Luciomelo1407** [reply]
thanks
